base/all/data/pecan_version_history.csv

```diff
@@ -1,51 +1,54 @@
-"PEcAn.photosynthesis",NA,NA,NA,NA,NA,1.2.6,1.3.1,1.3.3,1.3.3,1.3.6,1.3.7,1.3.6,1.4.0,1.4.1,1.4.2,1.4.3,1.4.4,1.4.5,1.4.6,1.4.7,1.4.8,1.4.9,1.4.10,1.4.10.1,1.5.0,1.5.1,1.5.2,1.5.3,1.6.0,1.7.0,1.7.1,1.7.2
-"PEcAn.PRELES",NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,1.4.4,1.4.5,1.4.6,1.4.7,1.4.8,1.4.9,1.4.10,1.4.10.1,1.5.0,1.5.1,1.5.2,1.5.3,1.6.0,1.7.0,1.7.1,1.7.2
-"PEcAn.priors",NA,NA,NA,NA,NA,1.2.6,1.3.1,1.3.3,1.3.3,1.3.3,1.3.7,1.3.3,1.4.0,1.4.1,1.4.2,1.4.3,1.4.4,1.4.5,1.4.6,1.4.7,1.4.8,1.4.9,1.4.10,1.4.10.1,1.5.0,1.5.1,1.5.2,1.5.3,1.6.0,1.7.0,1.7.1,1.7.2
-"PEcAn.qaqc",NA,NA,NA,NA,NA,1.0,1.3.1,1.3.3,1.3.3,1.3.3,1.3.7,1.3.3,1.4.0,1.4.1,1.4.2,1.4.3,1.4.4,1.4.5,1.4.6,1.4.7,1.4.8,1.4.9,1.4.10,1.4.10.1,1.5.0,1.5.1,1.5.2,1.5.3,1.6.0,1.7.0,1.7.1,1.7.2
-"PEcAn.remote",NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,1.5.1,1.5.2,1.5.3,1.6.0,1.7.0,1.7.1,1.7.2
-"PEcAn.rtm",NA,NA,NA,1.2.5,1.2.6,1.2.6,1.3.1,1.3.3,1.3.3,1.3.3,1.3.7,1.3.3,1.4.0,1.4.1,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
-"PEcAn.settings",NA,NA,NA,NA,NA,NA,1.3.1,1.3.3,1.3.3,1.3.3,1.3.7,1.3.3,1.4.0,1.4.1,1.4.2,1.4.3,1.4.4,1.4.5,1.4.6,1.4.7,1.4.8,1.4.9,1.4.10,1.4.10.1,1.5.0,1.5.1,1.5.2,1.5.3,1.6.0,1.7.0,1.7.1,1.7.2
-"PEcAn.SIPNET",NA,NA,NA,1.2.5,1.2.6,1.2.6,1.3.1,1.3.3,1.3.3,1.3.3,1.3.7,1.3.3,1.4.0,1.4.1,1.4.2,1.4.3,1.4.4,1.4.5,1.4.6,1.4.7,1.4.8,1.4.9,1.4.10,1.4.10.1,1.5.0,1.5.1,1.5.2,1.5.3,1.6.0,1.7.0,1.7.1,1.7.2
-"PEcAn.STICS",NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,1.7.2
-"PEcAn.uncertainty",NA,NA,NA,1.2.5,1.2.6,1.2.6,1.3.1,1.3.3,1.3.3,1.3.3,1.3.7,1.3.3,1.4.0,1.4.1,1.4.2,1.4.3,1.4.4,1.4.5,1.4.6,1.4.7,1.4.8,1.4.9,1.4.10,1.4.10.1,1.5.0,1.5.1,1.5.2,1.5.3,1.6.0,1.7.0,1.7.1,1.7.2
-"PEcAn.utils",NA,NA,NA,1.2.5,1.2.6,1.2.6,1.3.1,1.3.3,1.3.3,1.3.3,1.3.7,1.3.3,1.4.0,1.4.1,1.4.2,1.4.3,1.4.4,1.4.5,1.4.6,1.4.7,1.4.8,1.4.9,1.4.10,1.4.10.1,1.5.0,1.5.1,1.5.2,1.5.3,1.6.0,1.7.0,1.7.1,1.7.2
-"PEcAn.visualization",NA,NA,NA,1.2.5,1.2.6,1.2.6,1.3.1,1.3.3,1.3.3,1.3.3,1.3.7,1.3.3,1.4.0,1.4.1,1.4.2,1.4.3,1.4.4,1.4.5,1.4.6,1.4.7,1.4.8,1.4.9,1.4.10,1.4.10.1,1.5.0,1.5.1,1.5.2,1.5.3,1.6.0,1.7.0,1.7.1,1.7.2
-"PEcAn.workflow",NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,1.5.3,1.6.0,1.7.0,1.7.1,1.7.2
-"pecanapi",NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,1.7.0,1.7.1,NA
-"PEcAnRTM",NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,1.4.2,1.4.3,1.4.4,1.4.5,1.4.6,1.4.7,1.4.8,1.4.9,1.4.10,1.4.10.1,1.5.0,1.5.1,1.5.2,1.5.3,1.6.0,1.7.0,1.7.1,1.7.2
+"package","v1.0","v1.0.1","v1.1","v1.2.5","v1.2.6","v1.3","v1.3.2","v1.3.4","v1.3.5","v1.3.6","v1.3.6.1","1.3.7","v1.4.0","v1.4.1","v1.4.2","v.1.4.3","v1.4.4","v1.4.5","1.4.6","1.4.7","v1.4.8","v1.4.9","v1.4.10","v1.4.10.1","v1.5.0","v1.5.1","v1.5.2","v1.5.3","v1.6.0","v1.7.0","v1.7.1","v1.7.2","v1.8.0"
```

Comment:
It's hard to read in GitHub's review view, but the rightmost column of this block is your best summary of all the package versions in one place. If I did it right, this should agree with my proposal in https://gist.github.com/infotroph/f3b21b7500f24c1d13aa7617acd1776f